vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/model/SequenceActionType.java

```diff
@@ -129,6 +132,27 @@ public Type getArgumentType()
             return argumentType;
         }
     },
+    PRESS_KEYS_SIMULTANEOUSLY(false)
+    {
+        private final Type argumentType = TypeUtils.parameterize(List.class, String.class);
+
+        @Override
+        public void addAction(Actions actions, Object argument)
+        {
+            perform(argument, (List<String> arg) -> {
+                LinkedList<String> linkedArg = new LinkedList<>(arg);
```

Comment:
why do you need LinkedList here? if list is empty, null returned by pollFirst will not be suitable for keyDown

Reply:
I want to use the "LinkedList" for managing the base list and getting and removing the first element simultaneously. If the list will be empty or null I expect the same result as was mentioned in previous [https://github.com/vividus-framework/vividus/pull/833#discussion_r472916456](comment)

Reply:
But considering the "DeepCode" issue, I think I will continue to find another solution.

Reply:
refactored